pandas-stubs/core/strings.pyi

```diff
@@ -58,7 +63,7 @@ class StringMethods(NoNewAttributesMixin, Generic[T, _TS, _TM, _TS2]):
     @overload
     def cat(
         self,
-        others: Series | pd.Index | pd.DataFrame | np.ndarray | list[Any],
+        others: UnknownIndex | pd.DataFrame | npt.NDArray[Any] | list[Any],
```

Comment:
Shouldn't `UnknownSeries` also be in here?

Reply:
yup, thanks!